electron/legendary/games.ts

```diff
@@ -589,7 +589,8 @@ class LegendaryGame extends Game {
         gameInfo.app_name,
         ...exeOverrideFlag,
         offlineFlag,
-        launchArguments
+        launchArguments,
+        gameSettings.launcherArgs
```

Comment:
Does `gameSettings.launcherArgs` not exist under windows?

Reply:
Not sure what you mean, the `if (!isWindows)` block ends at line 584. These command parts are always set, regardless of the OS we're on